app/src/main/java/com/pluu/webtoon/data/daum/DaumDetailApi.kt

```diff
@@ -0,0 +1,100 @@
+package com.pluu.webtoon.data.daum
+
+import com.pluu.kotlin.iterator
+import com.pluu.webtoon.data.DetailRequest
+import com.pluu.webtoon.data.IRequest
+import com.pluu.webtoon.data.REQUEST_METHOD
+import com.pluu.webtoon.data.impl.AbstractDetailApi
+import com.pluu.webtoon.di.NetworkUseCase
+import com.pluu.webtoon.item.*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Pluu/WebToon/blob/e6d7b5dbb11d2d4fb917c63fbb0187d6361e5aea/app/src/main/java/com/pluu/webtoon/data/daum/DaumDetailApi.kt#L9"></span>Wildcard import (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>